front-end/src/utils/NotifyMe.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { ColoredButton } from '../AdminBar/adminStyles'
+import { NotifyContainer } from './styles'
+
+class NotifyMe extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {input: '', acceptsNotis: false, sentNotis: false, waitingFor: null, hide: false}
```

Comment:
WDYT about using the english term here, i.e. `acceptsNotification`? :)

Reply:
That'd be better, since the rest of the code is english